### DISCUSSION TITLE
module name i.e. got and search engine functionality

### DISCUSSION THREAD

**Discussion by gracilsinfor** [sindresorhus/got#2290] [Ideas]
hi, 
i am trying to use got package instead of request module and as expected some difficulties are found
my point is, got documentation, is written in english with a quite high level of abstraction demanding some advanced knowledge of node.js architecture but specially of got own complexity upon which given examples are not always easy to understand for beginner and even intermediate developer
now, is when using the help of community answers to trivial question is a gift and search engine an important tool that becomes useless by the therm to search which, as an english verb form, messes up the all outcome of the search.
i am not saying the name is this or is that i just find it complicates something it should not
perhaps gotjs or got-get-it or got-http or something else could provide a better interaction with search engines, or perhaps this isn't important at all
anyway thank you for your time

**Comment by sindresorhus** [comment]
I would recommend using ChatGPT for questions like this. Here is an example prompt:

> With Got (the Node.js HTTP client), how do I do x?

**Comment by gracilsinfor** [comment]
of course, I guess I am still attached to old uses 😂
thanks for the suggestion


A sábado, 5/08/2023, 11:33, Sindre Sorhus ***@***.***>
escreveu:
